src/v2/Apps/IdentityVerification/IdentityVerificationApp/index.tsx

```diff
@@ -31,7 +31,12 @@ const IdentityVerificationApp: React.FC<Props> = ({ me, relay }) => {
   const [showErrorModal, setShowErrorModal] = useState(false)
   const { trackEvent } = useTracking()
   if (!identityVerification || identityVerification.userID !== me.internalID) {
-    return <ErrorPage code={404} />
+    return (
+      <ErrorPage
+        code={400}
+        message={`You’re currently logged in as ${me.email}. To complete identity verification, please log out of this account, and log back into the account that received the email.`}
```

Comment:
Would it make sense to just return a very simple page wrapper with text instead of a "400" page?

Reply:
Do you have an example of that? I didn't know how to do a simple page wrapper.

Reply:
I'll adapt the current page so we don't have such a scary error.